pages/admin/request/create-new.tsx

```diff
@@ -0,0 +1,845 @@
+import PermitHolderTypeahead from '@components/admin/permit-holders/Typeahead';
+import SelectedPermitHolderCard from '@components/admin/requests/create/SelectedPermitHolderCard';
+import PermitHolderInformationForm from '@components/admin/requests/permit-holder-information/Form';
+import PhysicianAssessmentForm from '@components/admin/requests/physician-assessment/Form';
+import DoctorInformationForm from '@components/admin/requests/doctor-information/Form';
+import GuardianInformationForm from '@components/admin/requests/guardian-information/Form';
+import AdditionalQuestionsForm from '@components/admin/requests/additional-questions/Form';
+import PaymentDetailsForm from '@components/admin/requests/payment-information/Form';
+import BackToSearchModal from '@components/admin/requests/create/BackToSearchModal';
+import CancelCreateRequestModal from '@components/admin/requests/create/CancelModal';
+
+import { authorize } from '@tools/authorization';
+import { PhysicianAssessment } from '@tools/admin/requests/physician-assessment';
+import { DoctorFormData } from '@tools/admin/requests/doctor-information';
+import { GuardianInformation } from '@tools/admin/requests/guardian-information';
+import { RequestFlowPageState } from '@tools/admin/requests/types';
+import { PaymentInformationFormData } from '@tools/admin/requests/payment-information';
+import { AdditionalInformationFormData } from '@tools/admin/requests/additional-questions';
+import { NewApplicationPermitHolderInformation } from '@tools/admin/requests/permit-holder-information';
+import {
+  CreateNewApplicationRequest,
+  CreateNewApplicationResponse,
+  CREATE_NEW_APPLICATION_MUTATION,
+  GetApplicantNewRequestInfoRequest,
+  GetApplicantNewRequestInfoResponse,
+  GET_APPLICANT_NEW_REQUEST_INFO_QUERY,
+} from '@tools/admin/requests/create-new';
+import { useRouter } from 'next/router';
+import { formatDateYYYYMMDD } from '@lib/utils/format';
+
+/** Create New APP page */
+export default function CreateNew() {
+  const [step, setStep] = useState<RequestFlowPageState>(
+    RequestFlowPageState.SelectingPermitHolderPage
+  );
+  const [permitHolderExists, setPermitHolderExists] = useState(true);
+
+  const [applicantId, setApplicantId] = useState<number | null>(null); // RCD user id
+
+  // General information
+  const [permitHolderInformation, setPermitHolderInformation] =
+    useState<NewApplicationPermitHolderInformation>({
+      firstName: '',
+      middleName: '',
+      lastName: '',
+      dateOfBirth: '',
+      gender: 'MALE',
+      otherGender: '',
+      email: '',
+      phone: '',
+      receiveEmailUpdates: false,
+      addressLine1: '',
+      addressLine2: '',
+      city: '',
+      postalCode: '',
+    });
+  // Physician assessment
+  const [physicianAssessment, setPhysicianAssessment] = useState<PhysicianAssessment>({
+    disability: '',
+    patientCondition: null,
+    permitType: null,
+    physicianCertificationDate: new Date().toLocaleDateString('en-CA'),
```

Comment:
```suggestion
    physicianCertificationDate: '',
```
Initial value should be empty